platforms/axplat-riscv64-qemu-virt/src/irq.rs

```diff
@@ -144,4 +165,22 @@ impl IrqIf for IrqIfImpl {
             }
         )
     }
+
+    /// Sends Software Generated Interrupt (SGI)(s) (usually IPI) to the given dest CPU or all CPUs.
+    fn send_ipi(irq_num: usize, target: IpiTarget) {
+        match target {
+            IpiTarget::Other { cpu_id } => {
+                let res = send_ipi(HartMask::from_mask_base(1, cpu_id));
```

Comment:
Change `send_ipi` to `sbi_rt::send_ipi` to avoid confusion with this `send_ipi` function.